django_netjsongraph/tests/base/test_topology.py

```diff
@@ -332,7 +332,7 @@ def test_multiple_receive_removed(self):
         self.assertEqual(self.node_model.objects.count(), 3)
         self.assertEqual(self.link_model.objects.count(), 2)
         # expiration_time has not expired
-        self.assertEqual(self.link_model.objects.filter(status='down').count(), 0)
+        self.assertEqual(self.link_model.objects.filter(status='down').count(), 1)
```

Comment:
Also, this is causing a failure in the builds. @Rishabh2801 any reason for this change?

Reply:
There was an assertion error that 1!=0 so i changed it to 1

Reply:
I see the otherwise in the travis builds now. https://travis-ci.org/netjson/django-netjsongraph/jobs/448664710

Reply:
ok let me run test locally again and check

Reply:
Locally with this change it's showing me no errors
![2018-10-31-105249_1824x984_scrot](https://user-images.githubusercontent.com/34177647/47766866-304c2a00-dcfb-11e8-88f4-30f2200c8ec1.png)

Reply:
When the change is not applied
![2018-10-31-110407_1824x984_scrot](https://user-images.githubusercontent.com/34177647/47767225-e6644380-dcfc-11e8-9164-a99de96c6df7.png)

Reply:
@nemesisdesign it is also failing if i do not upgrade it and is giving me same assertion error